src/dropdown/dropdown.js

```diff
@@ -189,7 +189,7 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
 
   scope.$watch('isOpen', function(isOpen, wasOpen) {
     if (appendTo && self.dropdownMenu) {
-      var pos = $position.positionElements($element, self.dropdownMenu, 'bottom-left', true),
+      var pos = $position.positionElements($element, self.dropdownMenu, 'auto bottom-left', true),
```

Comment:
What is the rationale for this change? This changes the default for all apps using this component.